notebooks/integrating_auth_mast_service.ipynb

```diff
@@ -0,0 +1,455 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**This notebook serves to demonstrate how to integrate the `auth.mast` service into a Django web application.  The code was adopted from the solution derived for the `jwql` web application.  Much of the code was written by Christian Mesh of the Archive Science Branch.  Thank you Christian!**\n",
+    "\n",
+    "**Please note that this notebook does not work right out of the box, and some of the cells will not run without error.  There are various tokens and metadata that must be provided, and the code within is designed for a specific project.  It may be beneficial to structure the code in a different way for different applications.  This notebook simply serves as a guide.**\n",
```

Comment:
Are you actually able to run these functions in the notebook? It seems more to me like a collection of functions and things to integrate into a web app, but that can't be run here in this notebook format. (I got an error from the 4th cell that seems to be related to the function not existing inside the scope of a web app, for instance.) If I haven't misinterpreted the notebook, I think it might be helpful to be a bit clearer about that in the intro section up top: that the notebook isn't meant to be run really at all, but rather to serve as a reference for how to set up authentication within a web app. Does that make sense/sound reasonable?

Reply:
@laurenmarietta Yes, that makes sense, and I can add some more text here to make it more clear -- this notebook is merely a reference, not an operational piece.